### PR TITLE
[BACKLOG-38704] - (10.X) -Enable additional logging for Observability

### PR DESCRIPTION
**Pull request by lsaikrishna89**

### CODE REVIEW

File: engine/src/main/java/org/pentaho/di/trans/Trans.java [hunk 1]
```diff
@@ -150,7 +150,8 @@
 import org.pentaho.di.www.StartExecutionTransServlet;
 import org.pentaho.di.www.WebResult;
 import org.pentaho.metastore.api.IMetaStore;
```
Comment (peterrinehart) [inline engine/src/main/java/org/pentaho/di/trans/Trans.java:152]: Fix the copyright date in this file, please.

**Review by peterrinehart — CHANGES_REQUESTED** [review]
Please ensure this PR is linked to a jira ticket with a clear explanation of the reason for these changes.

**Comment by buildguy** [comment]
### :warning: Build finished in 1h 8m 36s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl engine,ui
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.di.trans.steps.metainject.MetaInjectTest.testMetastoreIsSet</b> (click to expand)</summary><p>

```
Central Log Store is not initialized!!!
```

</p></details>

  
Tests run: **4092**, Failures: **1**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9064/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :warning: Build finished in 1h 3m 37s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl engine,ui
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.di.trans.steps.metainject.MetaInjectTest.testMetastoreIsSet</b> (click to expand)</summary><p>

```
Central Log Store is not initialized!!!
```

</p></details>

  
Tests run: **4092**, Failures: **1**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9064/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :warning: Build finished in 1h 2m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl engine,ui
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.di.trans.steps.metainject.MetaInjectTest.testMetastoreIsSet</b> (click to expand)</summary><p>

```
Central Log Store is not initialized!!!
```

</p></details>

  
Tests run: **4092**, Failures: **1**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9064/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :warning: Build finished in 59m 23s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl engine,ui
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.di.trans.steps.metainject.MetaInjectTest.testMetastoreIsSet</b> (click to expand)</summary><p>

```
Central Log Store is not initialized!!!
```

</p></details>

  
Tests run: **4092**, Failures: **1**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9064/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1h 24s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
engine,plugins/meta-inject/impl,ui
```
---
:ok_hand: All tests passed!
 
Tests run: **4092**, Failures: **0**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9064/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: plugins/meta-inject/impl/src/test/java/org/pentaho/di/trans/steps/metainject/MetaInjectTest.java [hunk 2]
```diff
@@ -33,6 +33,7 @@
 import org.pentaho.di.core.exception.KettleValueException;
```
Comment (peterrinehart) [inline plugins/meta-inject/impl/src/test/java/org/pentaho/di/trans/steps/metainject/MetaInjectTest.java:33]: This file also needs the copyright header updated.

**Review by peterrinehart — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 59m 59s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
engine,plugins/meta-inject/impl,ui
```
---
:ok_hand: All tests passed!
 
Tests run: **4092**, Failures: **0**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9064/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 59m 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
engine,plugins/meta-inject/impl,ui
```
---
:ok_hand: All tests passed!
 
Tests run: **4092**, Failures: **0**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9064/7/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by lsaikrishna89** [comment]
closing the following PR as it has two commits unable to squash them, so I have opened the new PR
https://github.com/pentaho/pentaho-kettle/pull/9069

**Pull request closed without merging**